modules/node-shared/src/main/scala/org/tessellation/node/shared/infrastructure/snapshot/DataApplicationSnapshotAcceptanceManager.scala

```diff
@@ -174,10 +176,14 @@ object DataApplicationSnapshotAcceptanceManager {
           service.hashCalculatedState(newDataState.calculated)
         )
 
+        feeTransactions <- OptionT.liftF(
+          service.calculateFees(validatedUpdates)
```

Comment:
Not sure if `calculateFees` is a correct name as if I understand correctly the fees can only be retrieved, but they can't be created on the spot.

Reply:
@ryle-goehausen ☝️  thoughts?

Reply:
extractFees is more indicative I think, since they exist already and we're just pulling them from nested data

Reply:
Changed